### DISCUSSION TITLE
Can't laser grayscale don't have M4 or $$ 32

### DISCUSSION THREAD

**Discussion by vasjo1** [arkypita/LaserGRBL#1007] [Q&A]
Hello, 
I cannot select M4 anywhere to laser the gray scale in a photo, with me all lasing at the same strength. Also the $$ 32 I do not get in my settings but it goes to $$ 27. Have already reinstalled but it remains the same.
The program on the GRBL site under GRBL v1.1 is also no longer available.
Can you tell me how I solve this problem.
And my laser does not make a smooth movement when lasing, it goes shockingly to the left and to the right, only the rapid traverse goes smoothly.
Sorry for the bad English
Greetings from Belgium
Jozef

**Comment by StuartB4** [comment]
You can upgrade to GRBL 1.1 in LaserGRBL. It's on the Tools menu,  Flash Grbl Firmware.

![1](https://user-images.githubusercontent.com/16980026/82161937-bbb6ed00-9898-11ea-8edb-64ea47cb0fa4.jpg)
![2](https://user-images.githubusercontent.com/16980026/82161938-bc4f8380-9898-11ea-9720-70ae8d97f09f.jpg)
![3](https://user-images.githubusercontent.com/16980026/82161939-bce81a00-9898-11ea-8765-05ccf6ffd872.jpg)
![4](https://user-images.githubusercontent.com/16980026/82161940-bce81a00-9898-11ea-84e2-f9ebab5b8d73.jpg)
![5](https://user-images.githubusercontent.com/16980026/82161941-bd80b080-9898-11ea-886a-1cd384899c09.jpg)
![6](https://user-images.githubusercontent.com/16980026/82161943-bd80b080-9898-11ea-979e-d177438b6041.jpg)
![7](https://user-images.githubusercontent.com/16980026/82161944-be194700-9898-11ea-8fbf-cc2da2cef072.jpg)

**Comment by vasjo1** [comment]
Hello, 
thanks for the tips on how to install it, and now I can indeed print the photos as it should. 

Thanks and greetings from Belgium. 
Joseph 

Van: "Stuart" <notifications@github.com> 
Aan: "arkypita/LaserGRBL" <LaserGRBL@noreply.github.com> 
Cc: "vasjo" <vasjo@telenet.be>, "Author" <author@noreply.github.com> 
Verzonden: Maandag 18 mei 2020 00:47:38 
Onderwerp: Re: [arkypita/LaserGRBL] Can't laser grayscale don't have M4 or $$ 32 (#1007) 





You can upgrade to GRBL 1.1 in LaserGRBL. It's on the Tools menu, Flash Grbl Firmware. 

[ https://user-images.githubusercontent.com/16980026/82161937-bbb6ed00-9898-11ea-8edb-64ea47cb0fa4.jpg ] 
[ https://user-images.githubusercontent.com/16980026/82161938-bc4f8380-9898-11ea-9720-70ae8d97f09f.jpg ] 
[ https://user-images.githubusercontent.com/16980026/82161939-bce81a00-9898-11ea-8765-05ccf6ffd872.jpg ] 
[ https://user-images.githubusercontent.com/16980026/82161940-bce81a00-9898-11ea-84e2-f9ebab5b8d73.jpg ] 
[ https://user-images.githubusercontent.com/16980026/82161941-bd80b080-9898-11ea-886a-1cd384899c09.jpg ] 
[ https://user-images.githubusercontent.com/16980026/82161943-bd80b080-9898-11ea-979e-d177438b6041.jpg ] 
[ https://user-images.githubusercontent.com/16980026/82161944-be194700-9898-11ea-8fbf-cc2da2cef072.jpg ]

**Comment by StuartB4** [comment]
You're welcome. :+1:
